front/lib/resources/template_resource.ts

```diff
@@ -108,12 +109,24 @@ export class TemplateResource extends BaseResource<TemplateModel> {
     return this.visibility === "published";
   }
 
+  get pictureUrl() {
```

Comment:
:open_mouth: what does `get` do? :student:

Reply:
It's a getter: https://www.typescriptlang.org/docs/handbook/2/classes.html#getters--setters.